BuildDevint

```diff
@@ -17,12 +17,12 @@ BUILD_ECLIPSE=true
 BUILD_INTELLIJ=true
 INJECT_INTELLIJ_VERSION=false
 
-IJ_VERSION_MAJOR_BUILD=193.5233
-IJ_VERSION_MAJOR=2019.3
+IJ_VERSION_MAJOR_BUILD=201.3803.71
+IJ_VERSION_MAJOR=LATEST-EAP-SNAPSHOT
```

Comment:
The `\gradle.propertie` also needs to change.

Reply:
It's already changed in this PR. Check the source code at https://github.com/t-rufang/azure-tools-for-java/blob/eap201mvp/gradle.properties